plugins/kubebench/pkg/api.go

```diff
@@ -39,7 +40,10 @@ func KubeBenchAPI() {
 	log.Fatal(http.ListenAndServe(":80", mux))
 }
 
-func pluginAudit() {
+func pluginAudit(request *http.Request) {
```

Comment:
this method looks very similar to the same method in kubehunter and kubecapeplugin. can it be extracted to a common utility method?

Reply:
I don't think it is necessary. each plugin be independent